### PR TITLE
Add example app for authorization APIs

### PR DESCRIPTION
**Pull request by OleksiiZubko**
This is a simple web app with configured
compiling TS to JS.

App shows how to use APIs from `@here/olp-sdk-dataservice-api`
to work with Authorization Services.

Resolves: OLPEDGE-2374

Signed-off-by: Oleksii Zubko <ext-oleksii.zubko@here.com>

### CODE REVIEW

File: docs/examples/here-data-sdk-aaa-example/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+2) Add to your enviropment app key and secret.
+```
+export OLP_APP_KEY=<your app key>
+export OLP_APP_SECRET=<your app secret>
+```
+3) Install dependencies.
+
+```
+npm install
+```
```
Comment (HalynaDumych) [inline docs/examples/here-data-sdk-aaa-example/README.md:20]:
```suggestion
## Set up

1. Download the current folder to your PC.
2. Add your app key and secret to your enviropment.
    ```
    export OLP_APP_KEY=<your app key>
    export OLP_APP_SECRET=<your app secret>
    ```
3. Install the dependencies.

    ```
    npm install
    ```
```

File: docs/examples/here-data-sdk-aaa-example/README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Simple Authorization WEB App
```
Comment (HalynaDumych) [inline docs/examples/here-data-sdk-aaa-example/README.md:1]:
```suggestion
# Authorization Web App 
```

File: docs/examples/here-data-sdk-aaa-example/README.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+npm run build
+```
+
+3) Start web server.
+
+```
+npm start
+```
+
+4) Open in your favorite browser http://localhost:8080
```
Comment (HalynaDumych) [inline docs/examples/here-data-sdk-aaa-example/README.md:42]:
```suggestion
## Build and run

1. Compile TS files into JS files.

    ```
    npm run compile
    ```

2. Bundle with webpack.

    ```
    npm run build
    ```

3. Start the web server.

    ```
    npm start
    ```

4. Open the web server in your favorite browser: http://localhost:8080
```

File: docs/examples/here-data-sdk-aaa-example/README.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
+# Simple Authorization WEB App 
+
+This is a simple Web App with configured compiling TS to JS and bundling with webpack.
+The app shows how to use APIs from `@here/olp-sdk-dataservice-api`; 
+
+All function from AuthorizationAPI can be used in the same way as used in this example.
```
Comment (HalynaDumych) [inline docs/examples/here-data-sdk-aaa-example/README.md:6]:
```suggestion
It is a web app with configured compiling of TS files into JS files and bundling with webpack.
The app shows how to use the APIs from `@here/olp-sdk-dataservice-api`.

You can use all the `AuthorizationAPI` functions in the same way as in this example.
```

File: docs/examples/here-data-sdk-aaa-example/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+      "lib": [
+        "es2020",
+        "dom"
+      ],
+      "module": "commonjs",
+      "esModuleInterop": true,
+      "strict": true,
+    }
+  }
+
```
Comment (andescu) [inline docs/examples/here-data-sdk-aaa-example/tsconfig.json:16]: Same

File: docs/examples/here-data-sdk-aaa-example/README.md (hunk 3 above)
Comment (OleksiiZubko) [reply docs/examples/here-data-sdk-aaa-example/README.md:42]: Open the web server in - not correct, ignored that.

File: docs/examples/here-data-sdk-aaa-example/README.md (hunk 1 above)
Comment (OleksiiZubko) [reply docs/examples/here-data-sdk-aaa-example/README.md:20]: Set up - ignored, Setup should be used.

File: docs/examples/here-data-sdk-aaa-example/README.md (hunk 3 above)
Comment (HalynaDumych) [reply docs/examples/here-data-sdk-aaa-example/README.md:42]: Please change the formatting.

File: docs/examples/here-data-sdk-aaa-example/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+build
```
Comment (andescu) [inline docs/examples/here-data-sdk-aaa-example/.gitignore:1]: Is it not enough to have one .gitignore at the root level?
Why do we need a separate one?

Comment (OleksiiZubko) [reply docs/examples/here-data-sdk-aaa-example/.gitignore]: updated

**Review by andescu — APPROVED** [review]

**Review by HalynaDumych — APPROVED** [review]

**Pull request merged**